bugsnag-android-core/src/main/java/com/bugsnag/android/CachedThread.java

```diff
@@ -49,4 +49,24 @@ public void toStream(@NonNull JsonStream writer) throws IOException {
         }
         writer.endObject();
     }
+
+    long getId() {
```

Comment:
Are the CachedThread objects accessible in non-Bugsnag code?

Reply:
No - the class is package-private so should not be accessible to end-users. The only case where the object could be accessed is if someone writes a class in their application in the `com.bugsnag.android` package, or if they use Java's reflection API.